modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,29 +20,32 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
 |===
-||{product-title} 4.5 or earlier |{product-title} 4.6 or later
-|Stable {mtc-short} version a|{mtc-short} {mtc-version}._z_
+| |{product-title} 3.11 |{product-title} 4.0 to 4.5 |{product-title} 4.6 or later
```

Comment:
Add a column heading to describe the first column components. Otherwise, the table designs might indicate a formatting issues.

Reply:
@dymurray @stillalearner 
In the Details column I added an Installation row.
Are there any special Installation instructions for  OpenShift Container Platform 3.11